[ET][Portable][Build Size] Reduce build size of op_cumsum

manuelcandales opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #6021
* #6020
* #6019
* #6018
* #6017
* #6016
* #6015
* #6014
* #6013
* #6012
* #6011
* #6010
* #6009
* #6008
* #6007
* #6006
* #6005

98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

manuelcandales committed:
[ET][Portable][Build Size] Reduce build size of op_cumsum

98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_cumsum"


98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

[ghstack-poisoned]

malfet reviewed (approved):
But it can make code a bit slower, wouldn't it?

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_cumsum"


98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_cumsum"


98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable][Build Size] Reduce build size of op_cumsum"


98 K -> 9 K

Differential Revision: [D63997235](https://our.internmc.facebook.com/intern/diff/D63997235/)

[ghstack-poisoned]
